analyzer/src/main/kotlin/managers/Gradle.kt

```diff
@@ -135,10 +137,15 @@ class Gradle(
             }
         }
 
-        val gradleConnection = GradleConnector
+        val gradleConnector = GradleConnector
```

Comment:
Since the variable is called "connector" now, I'd like it a tad better to just assign it `GradleConnector.newConnector()` and ...

Reply:
Done.